app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt

```diff
@@ -84,7 +87,15 @@ class SettingsFragment : Fragment(), SettingsView, AdapterView.OnItemClickListen
                 shareIntent.putExtra(Intent.EXTRA_TEXT, shareSub)
                 startActivity(Intent.createChooser(shareIntent, getString(R.string.msg_share_using)))
             }
+            resources.getString(R.string.title_rate_us) -> startAppPlayStore()
+        }
+    }
 
+    private fun startAppPlayStore() {
+        try {
+            startActivity(Intent(Intent.ACTION_VIEW, getString(R.string.market_link).toUri()))
+        } catch (error: ActivityNotFoundException) {
+            startActivity(Intent(Intent.ACTION_VIEW, getString(R.string.play_store_link).toUri()))
```

Comment:
The first `startActivity(...)` can throw an exception but what about the second one? Could the `startActivity(Intent(Intent.ACTION_VIEW, getString(R.string.play_store_link).toUri()))` throw an exception too? 🤔

Reply:
I don't think startActivity(Intent(Intent.ACTION_VIEW, getString(R.string.play_store_link).toUri())) will throw an exception as it is a complete url to the app

Reply:
That's true! Since it is a web link it'll be fine. Btw, thanks for the answer.